### PR TITLE
fix(client): remove prism from app chunk

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Before:
Screen Shot 2021-05-24 at 1 29 44 PM

After:
Screen Shot 2021-05-24 at 1 28 23 PM
Screen Shot 2021-05-24 at 1 27 38 PM

### CODE REVIEW

**Comment by ShaunSHamilton** [comment]
Hey Ahmad,

There is a good chance this is just my run of GitPod, but I thought I would mention it. I cannot view the solutions (I do see Prism in the webpack internals):
image

I also see this error (likely nothing more than #39719 )
```console
Warning: Failed prop type: Invalid prop `files` of type `object` supplied to `SolutionViewer`, expected an array.
    in SolutionViewer (created by ProjectModal)
    in ProjectModal (created by CertificationSettings)
    in section (created by CertificationSettings)
    in CertificationSettings (created by withI18nextTranslation(CertificationSettings))
```

File: client/src/components/profile/components/TimeLine.js [hunk 1]
```diff
@@ -14,7 +14,9 @@ import { withTranslation } from 'react-i18next';
 import './timeline.css';
 import TimelinePagination from './TimelinePagination';
 import { FullWidthRow, Link } from '../../helpers';
-import SolutionViewer from '../../SolutionViewer/SolutionViewer';
+import Loadable from '@loadable/component';
+
+// import SolutionViewer from '../../SolutionViewer/SolutionViewer';
```
Comment (raisedadead) [inline client/src/components/profile/components/TimeLine.js:19]:
```suggestion
```

Drop the comment.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by ahmadabdolsaheb** [comment]
> Hey Ahmad,
> 
> There is a good chance this is just my run of GitPod, but I thought I would mention it. I cannot view the solutions (I do see Prism in the webpack internals):

That is interesting because I do not see the issue when developing and building locally.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

I fired up GitPod and the highlighting works as expected.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 
_Turns out all I needed to do was refresh the page_ - same thing happens on `.dev` 🤦

**Comment by raisedadead** [comment]
> Turns out all I needed to do was refresh the page - same thing happens on .dev 

Can you please elaborate, what you mean by "same thing happens on .dev"? This has not been shipped yet. Did you mean whatever issue you were reporting was pre-existing?

**Comment by ShaunSHamilton** [comment]
> Did you mean whatever issue you were reporting was pre-existing?

Yes. Oliver suggested it has to do with Redux not refreshing. What I experience is:
1) Complete a JS Project (Palindrome Checker)
2) Submit project
3) Navigate to `/settings`
4) Click on `Show Code` button in Certification section of settings page for submitted project
5) See modal appear saying "[no solution]"
6) Be sad, and bother Oliver. Refresh page
7) Click `Show Code` again
8) Modal appears with code - sadness diminishes

This happens on both a Gitpod instance of this PR, as well as on `.dev` for me. Does not happen on current production, but it would be most accurate to test with a new account which I have not.

**Comment by raisedadead** [comment]
We do not care about any developer environments, what I am interested in why would you have a difference in experience between staging and production.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**